.travis.yml

```diff
@@ -14,7 +14,11 @@ DEPLOY_TO_GITHUB: &DEPLOY_TO_GITHUB
     - PKGNAME="binaryen-$TRAVIS_TAG-$ARCH"
     - mv bin binaryen-$TRAVIS_TAG
     - tar -czf $PKGNAME.tar.gz binaryen-$TRAVIS_TAG
-    - sha256sum $PKGNAME.tar.gz > $PKGNAME.tar.gz.sha256
+    - if [ "$TRAVIS_OS_NAME" = "osx" ]; then
+        shasum -a 256 $PKGNAME.tar.gz > $PKGNAME.tar.gz.sha256;
```

Comment:
Does this one not also work on linux?

Maybe avoid the condition and add a comment as to why `shasum -a 256` is used rather than `sha256sum`?

Reply:
I don't think `shasum` is installed on Linux by default, but I'm also not entirely sure. I've never poked around Travis images too much and I recall having to do this historicsally.

As to adding a comment, done now!

Reply:
I ended up using python in both cases: https://github.com/WebAssembly/wabt/blob/master/scripts/sha256sum.py

Reply:
Ah.. yeah it looks like they come from completely different places:

```
$ dpkg -S /usr/bin/shasum
perl: /usr/bin/shasum
$ dpkg -S /usr/bin/sha256sum 
coreutils: /usr/bin/sha256sum
```

I think you can be pretty certain that perl is installed as part of the ubuntu core image that runs on travis.. but who knows if that `shasum` is that same as the mac one.  Do you know were it comes from on mac (`man shasum` should say)?

Reply:
Looks like the OSX one comes from perl as well, so presumably they're the same! A quick test shows they've got the same output, so seems reasonable to try it out and see what happens, updated now!